css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,18 @@
+button {
+  font-size: large;
+  padding: 10px 20px 10px 20px;
+}
+
+button:hover {
+  background-color: black;
+  color: white;
+}
+
+th:nth-child(n) {
+  background-color: black;
+  color: white;
+}
```

Comment:
The `nth-child` pseudo-class isn't necessary if you selected every single one. Also, there's one one.